Documentation update

bcamacho opened the pull request:
### Description
Added two new pages with instructions to connect a Ledger device to Celo Web Wallet & Celo Terminal

bcamacho committed:
Documentation update: Connecting Ledger to Celo Web Wallet & Celo Terminal

marekolszewski reviewed (dismissed):
This is great! What do you think of putting the CLI instructions on their own page as well and then linking to all three pages at the bottom of the current ledger page?

bcamacho committed:
Documentation update: Ledger wallet

bcamacho committed:
Documentation update: corrected error between pages from bad use of copy/paste

critesjosh committed:
Update connecting-ledger-celo-terminal-wallet.md

critesjosh commented on packages/docs/celo-holder-guide/connecting-ledger-celo-terminal-wallet.md:
```diff
@@ -0,0 +1,29 @@
+# Connect Ledger to Celo Terminal
```
I removed the `<br>` tags in this file, it was making the page render strangely.

bcamacho replied:
I used a local markdown editor which caused a conflict as GitHub renders different. The lesson I learned, verify with Github preview.

critesjosh commented on packages/docs/celo-holder-guide/connecting-ledger-celo-web-wallet.md:
```diff
@@ -0,0 +1,34 @@
+### Navigate to [https://celowallet.app/setup](https://celowallet.app/setup)
+</br>
+
+![https://Celowallet.app/setup](https://user-images.githubusercontent.com/2653576/130851740-742bb884-7ab7-465f-9d33-861ac6a5e909.png "image_tooltip")
```
It looks like the pictures are a little out of sync here. Should this picture be a part of step 2?

critesjosh commented on packages/docs/celo-holder-guide/connecting-ledger-celo-web-wallet.md:
```diff
@@ -0,0 +1,34 @@
+### Select “Use existing wallet”
+</br>
+
+![https://celowallet.app/setup/existing](https://user-images.githubusercontent.com/2653576/130851953-92ab2c36-01d3-493c-95d4-dce1d39f05fe.png "image_tooltip")
```
and this picture should be part of step 3

critesjosh commented on packages/docs/celo-holder-guide/connecting-ledger-celo-web-wallet.md:
```diff
@@ -0,0 +1,34 @@
+# Connect Ledger to Celo Web Wallet
+
+## Step 1
+</br>
```
suggest removing these breaks from this file as well.

marekolszewski commented on packages/docs/celo-holder-guide/ledger.md:
```diff
@@ -53,157 +52,8 @@ Press both buttons on the device at the same time to continue.
 
-## Setup Instructions
-### Install the Celo CLI
+#### Connect your Leger to Celo applications
```
After this heading, what do you think of an intro sentence like the following:

Now that your ledger is all set up, you can start using it with the following wallets:

bcamacho replied:
Great suggestion! Added the modification from your feedback.

bcamacho committed:
Documentation update: iterated from pull request feedback

bcamacho committed:
Documentation update: iterated from pull request feedback

bcamacho committed:
Documentation update: iterated from pull request feedback

bcamacho reviewed (commented):
Resolved errors from feedback with PR. @critesjosh please review.

critesjosh reviewed (approved):
lgtm!

bcamacho committed:
Update SUMMARY.md

Resolve merge conflict with summary.md by adjusting format for bullet points & removing comments

bcamacho committed:
resolved merge conflicts with branch

bcamacho committed:
resolved merge conflicts with branch

bcamacho committed:
resolved merge conflicts

bcamacho committed:
resolved reference link
